shell/plugins/steve/worker/index.js

```diff
@@ -4,9 +4,10 @@ import basicWorkerConstructor from '@shell/plugins/steve/worker/web-worker.basic
 import advancedWorkerConstructor from '@shell/plugins/steve/worker/web-worker.advanced.js';
 
 export const WORKER_MODES = {
-  WAITING:  'waiting',
-  BASIC:    'basic',
-  ADVANCED: 'advanced'
+  WAITING:      'waiting',
+  DESTROY_MOCK: 'destroy',
```

Comment:
I flip flopped with that one. If there were other changes i'd tweak this. In the interests of time though, and since it's approved, gonna mash the merge button